microsoft-365/compliance/add-your-organization-brand-to-encrypted-messages.md

```diff
@@ -148,6 +148,8 @@ To remove a custom branding template:
 
 ## Create an Exchange mail flow rule that applies your custom branding to encrypted emails
 
+[!IMPORTANT] If there is any 3rd party component that scans and modifies mail, it can prevent OME branding being applied correctly and shown to your recipients.
```

Comment:
```suggestion
> [!IMPORTANT]
> If there are any third-party components that scan and modify mail, it can prevent OME branding from being applied correctly and shown to your recipients.
```